pkg/virtctl/pause/pause.go

```diff
@@ -38,23 +38,20 @@ import (
 )
 
 const (
-	COMMAND_PAUSE   = "pause"
-	COMMAND_UNPAUSE = "unpause"
-	ARG_VM_SHORT    = "vm"
-	ARG_VM_LONG     = "virtualmachine"
-	ARG_VMI_SHORT   = "vmi"
-	ARG_VMI_LONG    = "virtualmachineinstance"
+	COMMAND_PAUSE = "pause"
+	ARG_VM_SHORT  = "vm"
+	ARG_VM_LONG   = "virtualmachine"
+	ARG_VMI_SHORT = "vmi"
+	ARG_VMI_LONG  = "virtualmachineinstance"
```

Comment:
Inline these or at least move them to the func where they are used in and don't make them exported.